README.md

```diff
@@ -1 +1,83 @@
-# s-covid
+# SCovid
+
+
+
+## Introduction
+
+**SCovid** is a Nodemcu based sensor for CO2 monitoring. 
+
+It has been developed mainly for teachers (educational environments), in order to facilitate the telemetry of CO2 level at school using [IoMBian OS](https://github.com/Tknika/iombian) project.
+![enter image description here](Sentsoreak.png "Sensors")
+
+It measures the forwarding magnitudes. 
+
+	 - CO2 level    pppm 
+	 - Tenperature  ºC 
+	 - Humidity     %
+	 - DEW point    ºC
+ 
+![enter image description here](Muntaia.png "Muntaia")
+
+
+## List of elements
+|Elements| Quantity | Link | 
+|---|---|---|
+| Sense Air S8| 1 |[Sense Air - Digi Key](https://www.digikey.es/product-detail/es/senseair-north-america-inc/004-0-0017/2194-004-0-0017-ND/10416536?utm_adgroup=Gas%20Sensors&utm_source=google&utm_medium=cpc&utm_campaign=Shopping_Product_Sensors%2C%20Transducers&utm_term=&productid=10416536&gclid=CjwKCAjwxuuCBhATEiwAIIIz0dKA79hlJd5p6Pi6lWrorZQlp4i2TtIozsbHxj0ZyZ9SqScUC76-VBoC6QgQAvD_BwE)|
+| DTH22 AM2302| 1 |[DTH22 - Electroson](https://electroson.com/producto/arduino-sensor-temperatura-y-humedad/ARDHT22)|
+| Nodemcu| 1 |[Nodemcu - Electroson](https://electroson.com/)|
+| Female-Female headers| 17 ||
+
+> **Warning**: Try to buy locally ```nearest store```.
+
+## Scheme
+![enter image description here](eskemafzz_bb.png "Muntaia")
+
+![enter image description here](eskemafzz_schem.png "Muntaia")
+
```

Comment:
- I would also add one more table to detail the relation between the sensors and NodeMCU pins. But this could be done later 😉

Reply:
There are those details on the very same squemm. Do you means those?

Reply:
I mean something like:

| NodeMCU | DHT22 | Sense Air S8 |
| -------------- | --------- | ------------------ |
|       3V3      |   VCC  |          -           |
|      GND     |   GND  |          -           |
|        D6      |    Data  |          -           |
|       Vin       |      -      |        VCC       |
|       GND    |      -      |        GND       |
|        RX      |      -      |         TX         |
|        TX      |       -     |         RX         |

Although... I have already done it for you... 🤦🏻